README.md

```diff
@@ -1 +1,157 @@
-# ruby-group-capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h2>📖 Ruby Group Capstone</h2>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🎥Presentation](#presentation)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [:question: FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Catalog of My Things <a name="about-project"></a>
+
+**Catalog of My Things:** This application is designed as a console utility enabling users to maintain a catalog of their possessions, encompassing a variety of categories such as books, music albums, movies, and games. The entire system is structured according to this [UML](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/images/catalog_of_my_things.png) class diagram
+
+## 🛠 Built With <a name="built-with"></a>
+
+  <ul>
+    <li>Ruby</li>
+  </ul>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- [x] **Ruby classes**
+- [x] **Ruby decorator**
+- [x] **Ruby associations**
+- [x] **Basic UI**
+- [x] **Preserve Data**
+- [x] **Unit Tests**
+- [x] **postgreSQL**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Presentation -->
+ 
+## 🎥 Video Presentation <a name="Presentation"></a>
+
+ -  [Video Link:](https://drive.google.com/file/d/1X8xE0Rpb4xXK7d5KF_2DAPOOHVQ02U-H/view?usp=sharing) 
```

Comment:
- Great job for Presenting all functionality, I think it would be better if you record a maximum of 5 minute video presentation to fit the [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things.md) 👍 
<img width="671" alt="2023-10-27_16-28" src="https://github.com/mershark/ruby-group-capstone/assets/52010789/8000e57a-c704-4771-962c-2cf5822e49a9">